ui/core/proto_utils/names.ts

```diff
@@ -207,7 +207,7 @@ export const pseudoStatNames: Map<PseudoStat, string> = new Map([
 	[PseudoStat.PseudoStatOffHandDps, 'Off Hand DPS'],
 	[PseudoStat.PseudoStatRangedDps, 'Ranged DPS'],
 	[PseudoStat.PseudoStatBlockValueMultiplier, 'Block Value Multiplier'],
-	[PseudoStat.TimewornBonus, 'Timeworn Pieces'],
+	[PseudoStat.TimewornBonus, 'Timeworn Pieces2'],
```

Comment:
Is this an accidental change?

Reply:
Yeah I'm not sure what made these changes but I'll revert them